api/src/test/java/org/openmrs/notification/SentMessageTest.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.notification;
+
+import java.util.Date;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SentMessageTest {
+	
+	@Test
+	public void SentMessage_shouldFillInAllParameters() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/openmrs/openmrs-core/file/2318749997/issues/source?bid=3204099&fileBranchId=3204129#l20)